ellocate.el

```diff
@@ -35,12 +35,19 @@
 (require 's)
 (require 'f)
 (require 'ivy)
+(require 'cl-lib)
 
-(defcustom ellocate-scan-dirs '(("~/" "~/ellocate-home-db"))
-  "An list of elements in this format: '(path database-location).
+(defgroup ellocate nil
+  "The locate command reimplemented in Emacs Lisp."
+  :prefix "ellocate-"
+  :group 'external
+  :link '(url-link :tag "GitHub" "https://github.com/walseb/ellocate"))
+
+(defcustom ellocate-scan-dirs '("~/" "~/ellocate-home-db")
+  "A list of elements in this format: '(path database-location).
 If the database field is nil, ellocate will not save the database to
 disk, but just store it in ram."
-  :type 'list
+  :type '(repeat string)
```

Comment:
If this is meant to be a tuple and not a repeating set of strings, you should probably use `'(list string string)`.